src/containers/SearchPage/SearchContainer.jsx

```diff
@@ -352,21 +354,11 @@ SearchContainer.defaultProps = {
   enabledTabs: [
     { value: 'all', name: 'all' },
     { value: 'topic-article', type: 'context-types', name: 'subject' },
-    {
-      value: 'urn:resourcetype:subjectMaterial',
+    ...allResourceTypes.map(resourceType => ({
```

Comment:
Since we are using all the resouceTypes the api provides we should'nt hardcode it anymore.